getting-started-spring-boot-maven/jqassistant/index.xml

```diff
@@ -0,0 +1,12 @@
+<!--1-->
+<jqassistant-rules
+        xmlns="http://schema.jqassistant.org/rule/v2.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://schema.jqassistant.org/rule/v2.0 https://schema.jqassistant.org/rule/jqassistant-rule-v2.0.xsd">
+
+    <!--2-->
+    <group id="Default">
+        <includeGroup refId="*:Default"/>
```

Comment:
I'm more a fan of explicitly including the groups. Please change to include maven:Default and package:Default

Reply:
It depends: agreeing with @StephanPirnbaum , please change to concrete groups here.